microsoft-edge/toc.yml

```diff
@@ -218,7 +218,11 @@
 
                 - name: Map preprocessed code to source code
                   href: devtools-guide-chromium/javascript/source-maps.md
-                  displayName: Sources tool
+                  displayName: Sources tool, source maps
+
+                - name: Debug original code using the Azure Artifacts symbol server
```

Comment:
Almost a better title (clearer) here in the TOC than top-of-page.   Pls try to explain in the title (hint) why "source maps" are of any interest to a webpage author.  Rule we've established for matching TOC title & top-of-page title: the TOC title is allowed to remove words, but otherwise, they need to be the same.

Other issue: is this article orphaned?  It seemed orphaned; not appearing in TOC.  Checking...